Feature/working branch

eypsrcnuygr opened the pull request:
- You can create new books with the necessary attributes.
- You can toggle the read status.
- You can delete them if you want.
- You can toggle the view to get the form or the table.

rOluochKe reviewed (changes requested):
Hello Team
### You have done well so far :clap: However, you have a few changes you need to work on:-
- [x] Remove unwanted files like .vscode from the project
- [ ] Separate dom manipulation form app logic
- [x] Write a detailed readme file
- [ ] We recommend you use es6 arrow functions to implement your code, see below
```
const clearView = () => {
  # Code
}
```
### Make the changes and submit for another code review. Good luck.
- Happy coding!

eypsrcnuygr commented:
- Since we can't reach the created HTML tags outside of the LoopTheArray function, we can't move togglebutton function, and DeleteButton function outside of it.
- Deleted the .vscode folder.
- Added a readme.
- Used arrow functions.

rOluochKe reviewed (approved):
Hello Team
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
### [Improvements]
- [ ] Upo saving the book record, make the modal close and clear fields for the next book.
- Happy coding!
